Cargo.toml

```diff
@@ -33,6 +33,7 @@ halo2_gadgets = { git = "https://github.com/snarkify/halo2", branch = "snarkify/
 bincode = "1.3"
 tempfile = "3.9.0"
 tracing-subscriber = "0.3.18"
+maplit = "1.0.2"
```

Comment:
one side note, when we introduce a new library for convenience, how would it affect the cargo build artifacts size?

Reply:
First of all, this is a dev dependency, i.e. it will affect only the test build.

Second, due to static linking, the size of the binary first of all changes depending on the code used, not the size of the code base of the imported crate.

Thirdly, it is usually a question of build time, because if the imported crate is large and the functionality is small, it is not worth it. There are only a couple of macros in the target crate, so it will not significantly affect the test build time.